.scripts/smoke-test.yml

```diff
@@ -9,9 +9,12 @@ jobs:
     strategy:
       matrix:
         Linux_Node18:
-          Pool: ${{ parameters.LinuxPool }}
+          Pool: azsdk-pool-mms-ubuntu-2004-general
```

Comment:
Do you not want to use to the new pool?

Reply:
I'm only touching this pipeline because I noticed it was using the "Azure Pipelines" pool, which should be reserved for Mac agents.  I'd rather wait to convert this whole repo to the new pool.